opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperTestDialog.tsx

```diff
@@ -25,6 +25,19 @@ interface CsvMapperTestDialogProps {
   configuration: string;
 }
 
+interface CsvMapperResult {
+  value: string;
+  nbRelationships: number;
+  nbEntities: number;
+}
+
+interface ResultCSVMapper {
```

Comment:
Not sure to undersand the use of that, shouldn't it be the type comming from relay ?

Reply:
In that case I think we need to modify relay configuration to handle JSON type the way we want in front.

Reply:
@richard-julien Any thougth ?

Reply:
Thanks to Julien, I find the correct way to handle the Stringify in the back.